media-sound/mpg321/mpg321-0.3.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A simple and lightweight command line MP3 Player"
+DESCRIPTION="mpg321 is a very popular command-line mp3 player. mpg321 is used \
+for frontends, as an mp3 player and as an mp3 to wave file decoder (primarily \
+for use with CD-recording software)."
+HOMEPAGE="http://mpg321.sourceforge.net/"
+COPYRIGHT="2001 Joe Drew
+	2006-2012 Nanakos Chrysostomos"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/mpg321/mpg321_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="056fcc03e3f5c5021ec74bb5053d32c4a3b89b4086478dcf81adae650eac284e"
+SOURCE_DIR="mpg321-$portVersion-orig"
+PATCHES="mpg321-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
```

Comment:
Can you please insert this block between SECONDARY_ARCHITECTURES and PROVIDES?
(You can copy it from the media-video/dvdauthor/dvdauthor-0.7.1.recipe.)
```
if [ "$targetArchitecture" != x86_gcc2 ]; then
	commandBinDir=$binDir
else
	commandBinDir=$prefix/bin
fi

```